clang/test/SemaHLSL/resource_binding_implicit.hlsl

```diff
@@ -14,6 +14,12 @@ RWBuffer<int> c;
 // No warning - explicit binding.
 RWBuffer<float> d : register(u0);
 
+// No warning - explicit binding.
+RWBuffer<float> dd : register(space1);
```

Comment:
Shouldn't there be a warning here?
Though the space param is being explicitly bound, the register number within that space is implicitly bound, no?

Reply:
Yes, you're correct. I've updated the latest, though this is commented out now because we can't actually write this test until #135287 goes in